warehouse/accounts/interfaces.py

```diff
@@ -97,6 +97,8 @@ def loads(token):
 
 
 class IPasswordBreachedService(Interface):
+    failure_message = Attribute("Password compromised")
```

Comment:
This should document what this attribute is, so something like ``Attribute("The message to describe the failure that occured")`` or something.